### ISSUE TITLE
Linking error when building from fresh checkout, libcarla_utils not found

### ISSUE THREAD

**Issue by SpotlightKid** [falkTX/Carla#1700]
```
[...]
Linking libcarla_frontend.so
/usr/bin/ld: cannot find -lcarla_utils: No such file or directory
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:189: ../../bin/libcarla_frontend.so] Error 1
make[1]: Leaving directory '/home/chris/work/aur/carla-git/src/carla/source/frontend'
make: *** [Makefile:189: frontend] Error 2
make: *** Waiting for unfinished jobs....
Linking libcarla_utils.so
```

Only on second build it succeeds.

**Comment by falkTX** [comment]
right yes, the new c++ frontend code needs the backend to be done first.
likely fixed in 3c10dcee953fb2af12eebac5578bc64f266ee0ec

**Comment by SpotlightKid** [comment]
Yes, looks good now.
